multi-node-efa-minimal.sh

```diff
@@ -87,6 +87,10 @@ for ID in ${INSTANCE_IDS[@]}; do
 done
 wait
 
+for i in $WORKSPACE/libfabric-ci-scripts/*_instance_status.sh; do
```

Comment:
Can you check the return code from `test_instance_status` instead? I don't see why a file has to be written to check the status.

Reply:
For multi-node job, test_instance_status is a background process. I could get the pid of each process and check the return status of each process. Was this what you meant? I just thought it was easier if test_instance_status fails, we just need to check whether the file exists, without checking for return code

Reply:
Why not just use `wait -n`?

Reply:
Rather than that, we can just check with something like this:

```
pids=""
RESULT=0

for ID in ${INSTANCE_IDS[@]}; do
   test_instance_status "$ID" &
   pids="$pids $!"
done

for pid in $pids; do
    wait $pid || let "RESULT=1"
done

if [ "$RESULT" == "1" ];
    then
       exit 65
fi
```